python/openassetio/log.py

```diff
@@ -75,7 +75,7 @@ def setSeverity(self, severity):
 
```

Comment:
[fix] Several lines above here (annoying github won't let me comment there) the `@name Filter Severity` docstrings can remove the "Confusingly" part now :partying_face:

Reply:
Ahh yes! Good spot, thanks :) removed in squish.